js/react/lib/components/level/level.component.tsx

```diff
@@ -0,0 +1,26 @@
+import { withAs } from "../../utils/hoc/with-as.hoc";
```

Comment:
Probably we need path aliases for this, like `@utils/hoc` or `@utils/tw-merge`

Reply:
Sure! I'll add path aliases in the next commit :)